FEXCore/Source/Utils/ArchHelpers/Arm64.cpp

```diff
@@ -2005,10 +2016,51 @@ HandleUnalignedAccess(FEXCore::Core::InternalThreadState* Thread, UnalignedHandl
     // Should not trigger - middle of an LDAXP/STAXP pair.
     LogMan::Msg::EFmt("Unhandled JIT SIGBUS STLXP: PC: 0x{:x} Instruction: 0x{:08x}\n", ProgramCounter, PC[0]);
     return NotHandled;
-  } else {
-    LogMan::Msg::EFmt("Unhandled JIT SIGBUS: PC: 0x{:x} Instruction: 0x{:08x}\n", ProgramCounter, PC[0]);
-    return NotHandled;
   }
+
+  // Check if another thread backpatched this instruction before this thread got here.
+  // Since we got here, this can happen in a couple situations:
+  // - Unhandled instruction (Shouldn't occur, programmer error)
```

Comment:
How would an unhandled instruction trigger SIGBUS / HandleUnalignedAccess?

Also, when talking about "programmer errors", it can be useful to stress which programmer you're talking about (FEX's or the guest application's?).

Reply:
Updated comment to state that it would be a FEX programmer error.